src/annotation.js

```diff
@@ -121,28 +132,19 @@ function updateElements(chart, state, options, mode) {
 		if (!el || !(el instanceof elType)) {
 			el = elements[i] = new elType();
 		}
-		const properties = calculateElementProperties(chart, el, annotation, elType.defaults);
-		animations.update(el, properties);
-
-		const display = typeof annotation.display === 'function' ? callCallback(annotation.display, [{chart, element: el}]) : valueOrDefault(annotation.display, true);
-		el._display = !!display;
+		el.options = merge(Object.create(null), [elType.defaults, annotation]);
```

Comment:
This is from the font merge. Should not set options directly, its going to skip animations.

Reply:
> Should not set options directly, its going to skip animations.

Does that mean that the options of element must be created by `animations.update();` ?
I have done in this way in order to do not have `_display` property on element but the `display` into options.

Reply:
Another doubt, @kurkle: I have seen the animation config but in my opinion the elements of this plugin are not animated.
Drawing the elements directly into plugin, how could they be animated?
Am I wrong?

Reply:
Take a look at the `animation.html` sample :)